aldryn_forms/contrib/email_validation/views.py

```diff
@@ -20,29 +20,30 @@ def validate_email(request, token):
     form_pk, email = unsigned_token.split('-', 1)
 
```

Comment:
While we're at it, we could also decorate the `validate_email` view with `@transaction.atomic` ?